tests/integration/bond_test.py

```diff
@@ -353,6 +355,73 @@ def test_changing_slave_order_keeps_mac_of_existing_bond(bond99_with_2_slaves):
     )
 
 
+@pytest.mark.tier1
+def test_adding_a_slave_keeps_mac_of_existing_bond(bond99_with_eth2, eth1_up):
+    bond_state = bond99_with_eth2[Interface.KEY][0]
+    bond_slaves = bond_state[Bond.CONFIG_SUBTREE][Bond.SLAVES]
+    bond_slaves.insert(0, ETH1)
+
+    current_state = statelib.show_only((BOND99,))
```

Comment:
nit: The bond name can also be extracted from the original state.

Reply:
Done, thank you!